### ISSUE TITLE
Trigger Record numbers start with 0 in v5 DAQ systems instead of 1

### ISSUE THREAD

**Issue by bieryAtFnal** [DUNE-DAQ/trigger#343]
I've noticed that the first Trigger Record number in each data taking run is zero.  Historically, the first first TR number has been 1, and I believe that we should not change this without a good reason.  In addition, I can imagine that the _art_ analysis framework might not like an event number of zero.  So, my sense is that we should fix this.

**Comment by MRiganSUSX** [comment]
I can confirm that this is a change that happened between v4 and v5 in the MLT. 

In both cases, the TR trigger number is set to be that of the `trigger_number` in the `TriggerDecision`:
- v4: https://github.com/DUNE-DAQ/dfmodules/blob/production/v4/plugins/TriggerRecordBuilder.cpp#L529
- v5: https://github.com/DUNE-DAQ/dfmodules/blob/develop/plugins/TRBModule.cpp#L568

MLT sets this TD `trigger_number`:
v4: https://github.com/DUNE-DAQ/trigger/blob/production/v4/plugins/ModuleLevelTrigger.cpp#L323
v5: https://github.com/DUNE-DAQ/trigger/blob/develop/plugins/MLTModule.cpp#L214
There is a simple '+ 1' that was removed (not my change). 
In both cases the counter `m_last_trigger_number` is initialized at 0. 

TLDR: this is a change that happened in MLT, and is easy to fix/change again. I don't know whether there was any reasoning when removing this +1.

**Comment by MRiganSUSX** [comment]
Seems like @glehmannmiotto made the change as part of this commit:
https://github.com/DUNE-DAQ/trigger/commit/1fb05dc84aaf76161a8d643683bf2047c8456fe1

**Comment by MRiganSUSX** [comment]
Fixed with https://github.com/DUNE-DAQ/trigger/pull/344
